exec/java-exec/src/test/java/org/apache/drill/test/FixtureBuilder.java

```diff
@@ -117,7 +122,27 @@ public FixtureBuilder configProperty(String key, Object value) {
     if (configProps == null) {
       configProps = defaultProps();
     }
-    configProps.put(key, value.toString());
+
+    if(value instanceof Collection<?>) {
```

Comment:
Can we just leave the collection in the single `configProps` object, and do the mapping when creating the final config? That is, for the DrillConfig, make the collection test when building DrillConfig. See comment above.